Sources/App/Extensions/Router+Extensions.swift

```diff
@@ -17,7 +17,7 @@ extension Request {
                 .first?
                 .split(separator: "-")
                 .first ?? "en")
-            let supportedLanguages = ["ar", "de", "en", "fa", "fr", "id", "ja", "ko", "pt", "ro", "ru", "tr", "zh", "nl", "el"]
+            let supportedLanguages = ["ar", "de", "en", "fa", "fr", "id", "ja", "ko", "pt", "ro", "ru", "tr", "zh", "nl", "el", "th"]
```

Comment:
Isn't this the exact same change as https://github.com/toptal/gitignore.io/pull/601/files#diff-f088ce64e1518b25971bf76a0a936cbd287599e47030a26dc5d77ba51f7bda43R20 ?
Why do we need both PRs?

Reply:
Thanks for suggestion #602, I should have done it twice due to a problem with my internet.